app_users/admin.py

```
@@ -112,6 +145,26 @@ class AppUserAdmin(GooeyModelAdmin):
+        return not lookup.startswith("password") and super().lookup_allowed(
+            lookup, value, request
+        )
+
+    def get_urls(self):
+        return [
+            path(
+                "<id>/password/",
+                self.admin_site.admin_view(self.user_change_password),
+                name="app_users_appuser_password_change",
+            ),
+        ] + super().get_urls()
```

Ruff (0.15.15)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)